### PR TITLE
Fix unsynchronized read

### PR DESCRIPTION
**Pull request by cberner**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java [hunk 1]
```diff
@@ -645,8 +648,10 @@ public void success(TaskInfo value)
                 try {
                     long currentRequestStartNanos;
                     synchronized (HttpRemoteTask.this) {
+                        //noinspection FieldAccessNotGuarded
```
Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java]: Why is this needed?

Comment (cberner) [inline presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java]: Because IntelliJ doesn't understand that "this" in the annotation is the same as "HttpRemoteTask.this"

Comment (dain) [inline presto-main/src/main/java/com/facebook/presto/server/remotetask/HttpRemoteTask.java]: you should file a bug report with Jetbrains and add it to a comment

**Comment by dain** [comment]
Looks good
